uwblueprint/community-fridge-kw
# style: Style select date time calendar

## Notion ticket link

[Style Select Date Time Calendar](https://www.notion.so/uwblueprintexecs/Update-Select-Date-Time-page-29b391eacc064d39a2d28dae01fd0e49)



## Implementation description
* Styled calendar (should be mobile-friendly and purple themed)
* Fixed issue where selecting Early Morning time slots are not saved to form
* Reset selected dayPart and timeslots when selecting a new date from datepicker



## Steps to test
1. Make sure you can go through the form, fill in the form values, go back and forth the pages without losing any selected values (and selected values must be correct)
2. After selecting a date, dayPart, and time slot, reselect a new date and all of those options should reset
3. Note: The calendar dates should be restricted to this week and next week (14 days total, Monday - Sunday)

https://user-images.githubusercontent.com/37950626/144726053-bffcbc80-d38f-4179-ab8c-1d3b9e686ecc.mov


## Checklist
